docs/RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Processes
+
+The following processes are necessary to actualize our releases. Each process has a *Cadence* on which it must execute and an *Responsible* that is responsible for autonomously doing so and reporting back any error in the RelEng<sup>1</sup> channel.
+
+## Mainline Release
+
+Cadence: every two weeks. Responsible: Release Team.
+
+This process aims to release the `release` branch as a *Mainline* release every two weeks. It needs to start three days before the proposed release deadline to allow leeway for unexpected issues. This process should eventually be automated.
```

Comment:
So the merge window is now three days and not a week?

Reply:
No. Two weeks minus three days is 11 days. But taking the "missing three days" from the previous release cycle into account, you end up with 14 days again.